e2e/dashboards-edit-v2-suite/dashboards-edit-variables.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { e2e } from '../utils';
+
+const PAGE_UNDER_TEST = 'kVi2Gex7z/test-variable-output';
+const DASHBOARD_NAME = 'Test variable output';
+
+describe('Dashboard edit variables', () => {
+  beforeEach(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+    cy.setLocalStorage('grafana.featureToggles', 'dashboardNewLayouts=true,kubernetesDashboards=true');
```

Comment:
you can do this in run-suite 
https://github.com/grafana/grafana/blob/main/e2e/run-suite#L117

Reply:
Tried that,  but [that is just referenced here ](https://github.com/grafana/grafana/blob/main/e2e/cypress/support/e2e.js#L54) and looks like only allows setting one feature toggle ( and overwrites any others that have been set ).

Reply:
If we want something like that, we could add 
```env[featureToggles]=kubernetesDashboards,dashboardNewLayouts ```

Then [here](https://github.com/grafana/grafana/blob/main/e2e/cypress/support/e2e.js#L54) we can add something to set it explicitly in local storage.

Reply:
Updated the global e2e [before each to allow using multiple toggles in the run-suite](https://github.com/grafana/grafana/pull/104775/files#diff-4f00b2b895254d4e8239777bf18843d681ac7b61674f7983a5481c8072425240R54-R72).

Reply:
merged [another pr](https://github.com/grafana/grafana/pull/104821) just to allow multiple toggles.